pom.xml

```diff
@@ -130,7 +131,7 @@
         <dependency>
             <groupId>org.projectlombok</groupId>
             <artifactId>lombok</artifactId>
-            <version>1.16.16</version>
+            <version>1.18.2</version>
```

Comment:
Is this required? We need to check if this does not create version conflict upstream.

Reply:
1.18.2 brings convenient @EqualsAndHashCode.Exclude, but if it could affect upstream, then I revert it back